tools/topology/topology2/avs-tplg/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Array of "input-file-name;output-file-name;comma separated pre-processor variables"
+set(TPLGS
+# CAVS HDMI only topology with passthrough pipelines
+"cavs-passthrough-hdmi\;sof-hda-generic-idisp\;USE_CHAIN_DMA=true"
```

Comment:
the source file cavs-passthrough-hdmi could be renamed, it's not limited to hdmi clearly.

This could be done separately in a different PR.

Reply:
done